objc/lib/TwitterText.h

```diff
@@ -103,6 +103,7 @@ FOUNDATION_EXTERN NSString * const kTwitterTextParserConfigurationV2;
 
 + (instancetype)defaultParser NS_SWIFT_NAME(defaultParser());
 + (void)setDefaultParserConfiguration:(TwitterTextConfiguration *)configuration;
+- (instancetype)initWithConfiguration:(TwitterTextConfiguration *)configuration;
```

Comment:
Thanks for this -- I am thinking I might want to make this the designated initializer (and also make a few other tweaks). I'll merge this (so you get credit!) but will be making some other minor updates soon as well.